packages/cli/package.json

```diff
@@ -60,6 +60,7 @@
     "@codeowners-flow/tsconfig": "workspace:*",
     "@types/node": "22.14.0",
     "@vitest/coverage-v8": "3.1.1",
+    "ajv": "8.17.1",
```

Comment:
pnpm and bun dependency resolution are different. Also how to install dependencies are different.